std_msgs/Header header  # note, match status messages with the commanded joint state using seq
string message
bool is_safe
